controllers/secret_controller.go

```diff
@@ -297,8 +297,8 @@ func createSubroutes(namespaceList []string, receiver receiverType) *alertmanage
 		// https://issues.redhat.com/browse/OSD-11298
 		// indications that master nodes have been terminated should be critical
 		// regex tests: https://regex101.com/r/Rn6F5A/1
-		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-[123]$"}, Match: map[string]string{"alertname": "MachineWithoutValidNode", "namespace": "openshift-machine-api"}},
-		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-[123]$"}, Match: map[string]string{"alertname": "MachineWithNoRunningPhase", "namespace": "openshift-machine-api"}},
+		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-.*[0-9]+$"}, Match: map[string]string{"alertname": "MachineWithoutValidNode", "namespace": "openshift-machine-api"}},
+		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-.*[0-9]+$"}, Match: map[string]string{"alertname": "MachineWithNoRunningPhase", "namespace": "openshift-machine-api"}},
```

Comment:
```suggestion
		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-.*[0-9]$"}, Match: map[string]string{"alertname": "MachineWithoutValidNode", "namespace": "openshift-machine-api"}},
		{Receiver: receiverCritical, MatchRE: map[string]string{"name": "^.+-master-.*[0-9]$"}, Match: map[string]string{"alertname": "MachineWithNoRunningPhase", "namespace": "openshift-machine-api"}},
```
curious about the `+`, is it needed?

Reply:
The + will ensure 1 or more number.  We don't need it today but we're hitting this bug because of assumptions of naming standard stability.  I figure hedging a little bit here is safe.